webviews/data_diff/data_diff.css

```
@@ -0,0 +1,104 @@
+@tailwind base;
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)